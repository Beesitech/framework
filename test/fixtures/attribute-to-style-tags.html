<!DOCTYPE html>
<html>
<head>
  <if condition="page.css">
    <style>{{{ page.css }}}</style>
  </if>
</head>
<body>
  <table class="text-xl" width="100%" height="600" bgcolor="#f0f0f0" background="https://example.com/image.jpg" align="left">
    <tr>
      <th valign="top">Test TH</th>
      <th width="200">Empty</th>
    </tr>
    <tr>
      <td width="600" height="600" bgcolor="#f0f0f0" background="https://example.com/image.jpg" align="center" valign="middle">
        <img src="https://example.com/image.jpg" width="300" height="200">
      </td>
    </tr>
  </table>
</body>
</html>
