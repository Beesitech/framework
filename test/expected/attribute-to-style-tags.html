<!DOCTYPE html>
<html>
<head>


  </head>
<body>
  <table class="text-xl" width="100%" height="600" bgcolor="#f0f0f0" background="https://example.com/image.jpg" align="left" style="font-size: 1.25rem; width: 100%" cellpadding="0" cellspacing="0" role="presentation">
    <tr>
      <th valign="top">Test TH</th>
      <th width="200" style="width: 200px">Empty</th>
    </tr>
    <tr>
      <td width="600" height="600" bgcolor="#f0f0f0" background="https://example.com/image.jpg" align="center" valign="middle" style="width: 600px">
        <img src="https://example.com/image.jpg" width="300" height="200" style="width: 300px" alt="">
      </td>
    </tr>
  </table>
</body>
</html>
